cog/daily_charge.py

```
@@ -23,95 +23,257 @@
+        if user.avatar is not None:  # 預設頭像沒有這個
+            embed.set_thumbnail(url=str(user.avatar))
+            embed.add_field(
+                name="您夠電了，明天再來!", value="⚡⚡⚡🛐🛐🛐", inline=False
+            )
+        return embed
+
+    def embed_successful(self, point, combo, user: discord.User | discord.Member):
         # 讀表符ID
         with open(
             f"{os.getcwd()}/DataBase/server.config.json", "r", encoding="utf-8"
         ) as file:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)